src/components/card/CompanyCard.js

```diff
@@ -23,12 +23,135 @@ class CompanyCard extends React.Component {
     this.setState({showModal: true});
   }
 
+  showLogo(link) {
+    if (link == undefined) {
+      return (<img className="logo" src='../styles/img/no-image-icon-23494.jpg'
+                   alt="logo"/>);
+    } else {
+      return (<img className="logo" src={link}
+                   alt="logo"/>);
+    }
+  }
+
+  joinList(elems) {
+    if (elems == undefined) return '';
```

Comment:
it might be safer to check falsey over here. with if((elemes) .... else ... [https://developer.mozilla.org/en-US/docs/Glossary/Falsy](url)